tsconfig.release.json

```diff
@@ -5,6 +5,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "sourceMap": false,
-    "declaration": true
+    "declaration": true,
+    "moduleResolution": "bundler"
```

Comment:
This chagne deserves a dedicated issue and description. I am unsure if it's needed at all. Let's revert it and verify if the packages are built properly.

Reply:
Done in b53e60f.